Dockerfile.assisted-installer-deployment

```diff
@@ -28,7 +28,9 @@ COPY dev-requirements.txt /tmp/dev-requirements.txt
 RUN pip install -r /tmp/dev-requirements.txt
 
 COPY requirements-rht.txt /tmp/requirements-rht.txt
+RUN git config --global http.sslVerify false
 RUN pip install -r /tmp/requirements-rht.txt || true
+RUN git config --global http.sslVerify true
```

Comment:
This is a hack (or a trick) to bypass the problem that underlying `pip install` runs `git clone` over HTTPS but we do not have this custom CA cert in the system store. As pips trusted-host parameter does not work when there is a nested git running, I did not find a smarter way to do what is being done here.